app/views/layouts/application.html.erb

```diff
@@ -11,8 +11,8 @@
   </head>
 
   <body>
-  <%= render 'shared/navbar' %>
-  <%= render 'shared/sidepanel' %>
+    <%= render 'shared/navbar' %>
```

Comment:
again why there is tab on line 14, 15
Please remove tab space and use 2 line space